### PR TITLE
minor fixes for pre-training

### PR DESCRIPTION
**Pull request by dkadochnikov**
normalization, linear layers and small fixes

### CODE REVIEW

File: src/models/pretraining.py [hunk 1]
```diff
@@ -20,26 +22,29 @@ def __init__(
...
-        self.linear = nn.Linear(self.emb_size, self.emb_size)
+        self.emb1 = nn.Linear(height * width, emb_size, bias=False)
+        self.letter_count = width // AVER_LETTER_WIDTH
+        self.linear = nn.Linear(emb_size, 2 * self.letter_count * height)
         self.decoder = nn.TransformerEncoder(
-            nn.TransformerEncoderLayer(d_model=self.emb_size, nhead=n_head, dim_feedforward=self.emb_size),
+            nn.TransformerEncoderLayer(d_model=emb_size, nhead=n_head, dim_feedforward=emb_size),
             num_layers=n_layers,
         )
+        self.emb2 = nn.Linear(emb_size, height * width, bias=False)
```
Comment (SpirinEgor) [inline src/models/pretraining.py:47]: Let's call this `head`

**Review by SpirinEgor — APPROVED** [review]
